### PR TITLE
Added support for Phy Pwr

### PR DESCRIPTION
**Pull request by DrBomb**
The [example guide](https://sautter.com/blog/ethernet-on-esp32-using-lan8720/) placed on the README.md file explains among other stuff the basics of hooking up the PHY to the esp32, and more importantly talks about the importance of using PHY PWR. Without it, IO0 will get input the oscillator's signal, usually triggering download mode on reset.

This PR adds support for PHY PWR. I used some of of the available example code for ethernet and cleaned up the the defines as well.

### CODE REVIEW

File: mos_esp32.yml [hunk 1]
```diff
@@ -10,6 +10,8 @@ config_schema:
   - ["eth.clk_mode", "i", 0, {title: "50 MHz clock source: 0 in <- GPIO0, 1 out -> GPIO0, 2 out -> GPIO16, 3 out -> GPIO17 (inverted)"}]
   - ["eth.mdc_gpio", "i", 23, {title: "GPIO to use for RMII MDC signal"}]
   - ["eth.mdio_gpio", "i", 18, {title: "GPIO to use for RMII MDIO signal"}]
+  - ["eth.enable_phy_pwr", "b", false, {title: "Enable PHY PWR control signal"}]
+  - ["eth.phy_pwr_gpio", "i", -1, {title: "GPIO to use for PHY PWR control signal"}]
```
Comment (rojer) [inline mos_esp32.yml:13]: well, if you have gpio dedicated to phy power you most likely want to use it. let's get rid of `enable_phy_pwr`

**Comment by rojer** [comment]
thanks for the patch, one small comment and we're good to go.
also, please clang-format with [this config](https://github.com/cesanta/mongoose-os/blob/master/.clang-format)

File: mos_esp32.yml (hunk 1 above)
Comment (DrBomb) [reply mos_esp32.yml:13]: Ok, I've removed it

**Comment by rojer** [comment]
ok, looks good now, integrating

**Pull request closed without merging**